certbot/plugins/manual.py

```diff
@@ -42,7 +42,9 @@ class Authenticator(common.Plugin):
 
 {validation}
 
-Once this is deployed,"""
+Before continuing, verify the record is deployed.
+
+The certificate authority will then be able to check the validity of your domain."""
```

Comment:
I'm sorry I didn't notice this before and that I'm continuing to nitpick this, but this sentence still isn't quite right in my opinion. The CA isn't checking the validity of your domain in the sense that the domain name itself is valid or invalid. The name provided by the user may be an acceptable and recognized domain, but the CA is checking that they have control over the domain by verifying they were able to configure a DNS record as requested. Here's two links for seeing how this is normally described in this context:

* https://tools.ietf.org/html/draft-ietf-acme-acme-06#section-8
* https://en.wikipedia.org/wiki/Domain-validated_certificate

I hope this makes sense and feel free to either update this sentence or delete it entirely. After that, I'll merge this PR!